ts/session/apis/file_server_api/FileServerApi.ts

```diff
@@ -130,30 +133,46 @@ const parseStatusCodeFromOnionRequestV4 = (
  * This call is onion routed and so do not expose our ip to github nor the file server.
  */
 export const getLatestReleaseFromFileServer = async (
-  userEd25519SecretKey: Uint8Array
-): Promise<string | null> => {
+  userEd25519SecretKey: Uint8Array,
+  releaseType?: ReleaseChannels
+): Promise<[string, ReleaseChannels] | null> => {
   const sigTimestampSeconds = NetworkTime.getNowWithNetworkOffsetSeconds();
   const blindedPkHex = await BlindingActions.blindVersionPubkey({
     ed25519SecretKey: userEd25519SecretKey,
   });
-  const signature = await BlindingActions.blindVersionSign({
+  const method = 'GET';
+  let releaseChannel = Storage.get('releaseChannel') as ReleaseChannels;
+
+  if (!releaseChannel) {
+    releaseChannel = 'latest';
+    await Storage.put('releaseChannel', releaseChannel);
+  }
```

Comment:
we should maybe sanitize this (i.e. default to latest if releaseChannel doesn't match a valid value